cdt_slugs.yaml

```diff
@@ -6,6 +6,11 @@ alsa-lib-devel:
 audit-libs:
   custom: false
   license_file: licenses/audit-libs-copying.lib
+audit-libs-devel:
+  custom: false
+  license_file: licenses/audit-libs-copying.lib
+  dep_remove:
+    - kernel-headers
```

Comment:
> Maybe the better question is, can you show me how to add a skipped dependency for an alma8 CDT?

This works for both cos7/alma8

Reply:
@beckermr, this case might actually justify some further functionality. My understanding is that `dep_replace` can only replace one CDT with _another_ CDT, but what we'd actually want here is to replace a CDT with a conda-package, because we do have `kernel-headers_<arch>` coming from https://github.com/conda-forge/linux-sysroot-feedstock. 

For now I've removed `kernel-headers` because even if `audit-libs-devel` depends on it, I didn't want to pull it into the alma8 allowlist. It's possible though that the package is non-functional without the headers.

Reply:
> which always pulls in the kernel headers.

Yeah, the kernel-headers are special in that they're a dep of the sysroot which is there already. I guess we can defer this question (i.e. a CDT that depends on another distro-package for which we have a conda equivalent) if and until it ever becomes relevant.